docs/DeveloperGuide.md

```diff
@@ -1,34 +1,833 @@
-# Developer Guide
+# WatchNext Developer Guide
```

Comment:
Overall a well structured DG which makes for a pleasant and easy read. However, there are some small tweaks that can be made to further improve your DG.